include/proto/kvs.proto

```diff
@@ -36,10 +42,21 @@ message SetValue {
 }
 
 message CausalValue {
-  map<uint32, uint32> vector_clock = 1;
+  map<string, uint32> vector_clock = 1;
   repeated string values = 2;
 }
 
+message Dependency {
```

Comment:
Plural.

Reply:
I guess this is fine? because each message has only one dependency

Reply:
Aha, okay.